test/run_test.py

```diff
@@ -24,6 +24,8 @@
 import torch.distributed as dist
 from torch.multiprocessing import current_process, get_context
 from torch.testing._internal.common_utils import (
+    dynamo_strict_counter,
```

Comment:
The change itself looks good to me, but there is an chicken-and-egg issue with this linter job https://github.com/pytorch/pytorch/blob/main/.github/workflows/lint.yml#L209-L232 smoke test for `run_test`.  To avoid the need to rebuild PyTorch, the job installs PyTorch from nightly, which atm doesn't have the new fields `dynamo_strict_counter` and `dynamo_total_counter`, so it fails with an import error https://hud.pytorch.org/pr/pytorch/pytorch/115284#19389428304.

I think a fix here could be to set `PYTHONPATH` in https://github.com/pytorch/pytorch/blob/main/.github/workflows/lint.yml#L232:

```
PYTHONPATH="${PWD}" python3 test/run_test.py --include test_vulkan --verbose
```

So the check correctly use `torch.testing._internal.common_utils` from the PR.

Reply:
What if we just import them lazily? Let me try that.